### DISCUSSION TITLE
gJ gS is the keymap for splitjoin.vim?

### DISCUSSION THREAD

**Discussion by scc02** [ecosse3/nvim#60] [Q&A] [answered]
I read the [README.md](https://github.com/ecosse3/nvim#readme), saw `Smartly joins lines based on treesitter`, but after read splitjoin.vim's README.md, seems not base on treesitter

**Comment by ecosse3** [comment]
Yeah, I see its in VimScript so it doesn't use treesitter, but anyway it works good. Will change the README in next release.

**Comment by ecosse3** [comment, marked as answer]
@ShiChenCong There is now a similar plugin that is using a tree-sitter for this. Didn't test yet since `splitjoin.vim` seems to work great all the time.

Checkout https://github.com/AckslD/nvim-trevJ.lua.

**Reply by scc02** [reply]
Thank you, [nvim-trevJ.lua](https://github.com/AckslD/nvim-trevJ.lua) works very well
